lib/generators/alchemy_i18n/install/install_generator.rb

```diff
@@ -55,7 +55,7 @@ def append_manifest
 
       def add_rails_i18n
         environment do
-          "config.i18n.available_locales = #{locales.map(&:to_sym).inspect}"
+          "config.i18n.available_locales = #{[:en].concat(locales.map(&:to_sym)).inspect}"
```

Comment:
Not all stores have English as locale necessarily.

I think we should use whatever the user wants as `locale`, but ignore `en` (from given `locales` option) whenever we add JS translations (because they are already build in):

here https://github.com/AlchemyCMS/alchemy_i18n/blob/main/lib/generators/alchemy_i18n/install/install_generator.rb#L24

here

https://github.com/AlchemyCMS/alchemy_i18n/blob/main/lib/generators/alchemy_i18n/install/install_generator.rb#L32

and here 

https://github.com/AlchemyCMS/alchemy_i18n/blob/main/lib/generators/alchemy_i18n/install/install_generator.rb#L43